prac_05/state_names.py

```diff
@@ -0,0 +1,25 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File has been reformatted
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+
```

Comment:
You could just use while state_code != "" and before that ask for an input so there is no need to have a break in it